templates/dell_powerconnect_show_interfaces_description.textfsm

```diff
@@ -0,0 +1,11 @@
+Value INTERFACE (\S+)
+Value DESCRIPTION (\S.*?)
```

Comment:
Does using `(\S*)` work? then that will simplify the regex match below?

Reply:
Hi @jmcgill298, Yes that work, but your suggest is more simply.